core/spec/models/spree/store_spec.rb

```diff
@@ -37,19 +37,32 @@
   end
 
   describe ".default" do
-    let!(:store)    { create(:store) }
-    let!(:store_2)  { create(:store, default: true) }
+    before do
+      Spree::Store.delete_all
```

Comment:
This is unnecessary. There aren't any stores being created before this (and we truncate the database at the start of the core test suite)

Otherwise :+1: